src/app/test-record/test-record.container.spec.ts

```diff
@@ -1,34 +1,92 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
-import { of } from 'rxjs';
-import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { INITIAL_STATE, Store } from '@ngrx/store';
-import { hot } from 'jasmine-marbles';
+import { BehaviorSubject, of } from 'rxjs';
+import {
+  Component,
+  CUSTOM_ELEMENTS_SCHEMA,
+  EventEmitter,
+  Input,
+  Output
+} from '@angular/core';
+import { Action, Store } from '@ngrx/store';
 import { TestRecordContainer } from '@app/test-record/test-record.container';
 import { SharedModule } from '@app/shared/shared.module';
-import { TestResultModel } from '@app/models/test-result.model';
 import { ActivatedRoute, convertToParamMap } from '@angular/router';
-import { TestType } from '@app/models/test.type';
-import { TestRecordMapper } from '@app/test-record/test-record.mapper';
+import { TestRecordMapper, TestTypesApplicable } from '@app/test-record/test-record.mapper';
+import { TestRecordTestType } from '@app/models/test-record-test-type';
+import { VIEW_STATE } from '@app/app.enums';
+import { TestResultTestTypeNumber } from '@app/models/test-result-test-type-number';
+import { TestResultModel } from '@app/models/test-result.model';
+import { EVehicleTestResultModelActions } from '@app/store/actions/VehicleTestResultModel.actions';
+import { getPreparers, getTestStations } from '@app/store/selectors/ReferenceData.selectors';
+import { map } from 'rxjs/operators';
+import {
+  getCurrentState,
+  selectTestTypeById
+} from '@app/store/selectors/VehicleTestResultModel.selectors';
+import { Preparer } from '@app/models/preparer';
+import { TestStation } from '@app/models/test-station';
+import { TESTING_TEST_MODELS_UTILS } from '@app/utils/testing-test-models.utils';
+
+const mockSelector = new BehaviorSubject<any>(undefined);
+
+class MockStore {
```

Comment:
Same this can be part of the same MockStore we have in the other file,
We can have under utils/ as this is reused and have Class for it, i believe whether this is test result or tech record we might show the data, it will only change based on the vehicle type.